src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -154,6 +138,41 @@ private async Task ProcessEpisodeDataAsync(IEnumerable<BssEpisode> episodes, str
         }
     }
 
+    private const string DateFormat = "dd/MM/yyyy";
+    private EpisodeDto ProcessEpisode(BssEpisode episode)
```

Comment:
Let's give this method a name that more accurately describes what it does. Something along the lines of MapEpisodeToEpisodeDto would be better